client/rest/rest.go

```diff
@@ -6,12 +6,17 @@ import (
 	"github.com/gorilla/mux"
 )
 
+// DeprecationURL is the URL for migrating deprecated REST endpoints to newer ones.
+// TODO Switch to `/` (not `/master`) once v0.40 docs are deployed.
+// https://github.com/cosmos/cosmos-sdk/issues/8019
+const DeprecationURL = "https://docs.cosmos.network/master/migrations/rest.html"
```

Comment:
Can we put `v0.40.0` here? It obviously won't exist until we publish the release, but at least it's less likely to forget about it.

Reply:
The idea was that people testing RCs right now (and seeing errors like in #8002 #8003) have something to look at

ref: https://github.com/cosmos/cosmos-sdk/pull/7992#issuecomment-732224347

Also added an item in https://github.com/cosmos/cosmos-sdk/issues/8019 to not forget about it.